base/intfuncs.jl

```diff
@@ -1075,3 +1075,22 @@ function binomial(n::T, k::T) where T<:Integer
     end
     copysign(x, sgn)
 end
+
+"""
+    iroot(n::Integer, r::Integer)
+computes floor(n^(1/r)) precisely
```

Comment:
Missing newline after function signature, missing code quoting of code, missing capital letter and period. Would also be good with a doctests showing the difference between `iroot` and the naive way of writing it.